src/readqps.jl

```diff
@@ -432,7 +438,7 @@ function read_quadobj_section(qps, varnames)
         pos = position(qps)
     end
     seek(qps, pos)  # backtrack to beginning of line
-    sparse(qrows, qcols, qvals, nvar, nvar)
+    return qrows, qcols, qvals
```

Comment:
We didn't use `return` at other places. Either add it everywhere or leave it out.